### DISCUSSION TITLE
Adding the ability to add coupons to the Demae channel

### DISCUSSION THREAD

**Discussion by pikapictured** [WiiLink24/food-server#31] [Ideas]
Sorry if this is stupid, I am new to WiiLink and don't really know how this works, and y'all are amazing for reviving old WiiLink24 channels. I just have a suggestion:

Would it possible to add coupons from the Domino's app to the Demae channel? Thanks! 🙏🏻

Something like this:
![Screenshot_20231009_192016_Domino's.jpg](https://github.com/WiiLink24/food-server/assets/77557847/5f35125b-8f33-4598-a657-8ec1f97493c0)

![Screenshot_20231009_191955_Domino's.jpg](https://github.com/WiiLink24/food-server/assets/77557847/a9c3afd8-afc2-4855-bdf3-3840c7dcfdc8)

**Comment by NecroNovaVGitHub** [comment]
I do not directly work on the Demae Server here, but i am here to inform you that it is IMPOSSIBLE to add coupons to the Demae Channel. Coupons do not exist on the Demae Channel as a feature, that feature is instead on Wii No Ma (aka Wii Room). And even then, coupons on Wii no ma are currently unusable in the current version of Wii no ma.

**Reply by pikapictured** [reply]
Thanks for the quick reply. Yeah a shame but I get it, it's almost impossible to get that, was just wondering. 🙏🏻
